internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/CoordinatorGrpcRetryableClient.java

```diff
@@ -64,7 +65,10 @@ public CoordinatorGrpcRetryableClient(
         ThreadUtils.getDaemonFixedThreadPool(heartBeatThreadNum, "client-heartbeat");
   }
 
-  public void sendAppHeartBeat(RssAppHeartBeatRequest request, long timeoutMs) {
+  @Override
+  public RssAppHeartBeatResponse sendAppHeartBeat(RssAppHeartBeatRequest request) {
```

Comment:
This method name looks just like a operation, rather than daemon thread to always sendAppHeartbeat.

From my sight, how about renaming this as the from `sendAppHeartBeat` to `scheduleAtFixedRateToSendAppHeartBeat` ?

Reply:
WDYT?

Reply:
This method seems only uses a thread pool to send heartbeats in parallel, may be we don't need to modify the name?

Reply:
It will make other developper misunderstand this that will call this method for multiple times.

Reply:
ok, I have modified the func name